recipes/flask-cache/meta.yaml

```diff
@@ -0,0 +1,39 @@
+{% set name = "flask-cache" %}
+{% set version = "0.13.1" %}
+{% set build = 0 %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version}}.tar.gz
+  url: https://pypi.python.org/packages/91/c4/f71095437bd4b691c63f240e72a20c57e2c216085cbc271f79665885d3da/Flask-Cache-0.13.1.tar.gz
```

Comment:
Can you change that to:

`https://pypi.io/packages/source/F/Flask-Cache/Flask-Cache-{{ version }}.tar.gz`